src/main/java/org/takes/rq/RqWithDefaultHeader.java

```diff
@@ -0,0 +1,68 @@
+/**
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2015 Yegor Bugayenko
```

Comment:
Right. But what about checkstyle violations? Build will fail if I change to 2016